README.rst

```diff
@@ -1,7 +1,210 @@
+that should be ignored by ``pytest-doctestplus`` but do not need to be ignored
+by other ``pytest`` features.
+
+Doctest Directives
+~~~~~~~~~~~~~~~~~~
+
+The ``pytest-doctestplus`` plugin defines `doctest directives`_ that are used
+to control the behavior of particular features. For general information on
+directives and how they are used, consult the `documentation`_. The specifics
+of the directives that this plugin defines are described in the sections below.
+
+.. _doctest directives: https://docs.python.org/3/library/doctest.html#directives
+.. _documentation: https://docs.python.org/3/library/doctest.html#directives
+
+Floating Point Comparison
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some doctests may produce output that contains string representations of
+floating point values.  Floating point representations are often not exact and
+contain roundoffs in their least significant digits.  Depending on the platform
+the tests are being run on (different Python versions, different OS, etc.) the
+exact number of digits shown can differ.  Because doctests work by comparing
+strings this can cause such tests to fail.
+
+To address this issue, the ``pytest-doctestplus`` plugin provides support for a
+``FLOAT_CMP`` flag that can be used with doctests.  For example:
+
+.. code-block:: none
+
+  >>> 1.0 / 3.0  # doctest: +FLOAT_CMP
+  0.333333333333333311
+
+When this flag is used, the expected and actual outputs are both parsed to find
+any floating point values in the strings.  Those are then converted to actual
+Python `float` objects and compared numerically.  This means that small
+differences in representation of roundoff digits will be ignored by the
+doctest.  The values are otherwise compared exactly, so more significant
+(albeit possibly small) differences will still be caught by these tests.
+
+Skipping Tests
+~~~~~~~~~~~~~~
+
+Doctest provides the ``+SKIP`` directive for skipping statements that should
+not be executed when testing documentation. However, it is often useful to be
+able to skip docstrings associated with particular functions, methods, classes,
+or even entire files.
+
+Skip Unconditionally
+^^^^^^^^^^^^^^^^^^^^
+
+The ``pytest-doctestplus`` plugin provides a way to indicate that certain
+docstrings should be skipped altogether. This is configured by defining the
+variable ``__doctest_skip__`` in each module where tests should be skipped. The
+value of ``__doctest_skip__`` should be a list of wildcard patterns for all
+functions/classes whose doctests should be skipped.  For example::
+
+   __doctest_skip__ = ['myfunction', 'MyClass', 'MyClass.*']
+
+skips the doctests in a function called ``myfunction``, the doctest for a
+class called ``MyClass``, and all *methods* of ``MyClass``.
+
+Module docstrings may contain doctests as well.  To skip the module-level
+doctests include the string ``'.'`` in ``__doctest_skip__``.
+
+To skip all doctests in a module::
+
+   __doctest_skip__ = ['*']
+
+Doctest Dependencies
```

Comment:
I don't see any mention of `.. doctest-requires:: scipy` in this section but it is used in `astropy`. Is that still supported?

Reply:
Good point. They should be included here.

Reply:
Actually, I'm not sure we should document these here quite yet since I think they also require astropy's sphinx plugins in order to work, and we haven't provided separate repositories for those yet.

Reply:
I opened an issue to track this in #13.